plugins/CorePluginsAdmin/angularjs/plugins/plugin-management.directive.js

```diff
@@ -25,6 +25,23 @@
 
                     var uninstallConfirmMessage = '';
 
+                    $('.uploadPlugin').click(function (event) {
```

Comment:
If easy to do then +1. Otherwise we can just add a comment in both files to note that the code is duplicated in the other file